### ISSUE TITLE
Player doesnt update palette after using sweet scent

### ISSUE THREAD

**Issue by Jaizu** [aarant/pokeemerald#22]
Please test in vanilla emerald too, I don't discard this being a me issue

![image](https://github.com/aarant/pokeemerald/assets/18596778/83d7b554-a97a-4b6a-be14-be797d322820)

**Comment by Jaizu** [comment]
Seems like this works with the vanilla pal tags but it doesnt with the one that I set for my sprite (0x1139), any idea why that might happen? Sweet scent seems to work fine, the issue is that the player pal doesnt get updated with the time.

**Comment by aarant** [comment]
Yeah I can't reproduce this and am not sure why the tag value would matter, looking at the code. Can you send a video or something?
